### DISCUSSION TITLE
Other representation after updating the page of GLuonTS/ Trainings and Test split in an other way

### DISCUSSION THREAD

**Discussion by MarleneBs** [awslabs/gluonts#2375] [Q&A]
I put my multiple_ts together as follows:

```py
multiple_ts =[dataframe1, dataframe 2,...]
```
and I did my training and testing SPlit as follows: 

```py
to_pandasdataset = lambda data: PandasDataset(
    multiple_ts,
    target="vakue1",
    feat_dynamic_real=feat,
)
train = to_pandasdataset([ts.iloc[:-prediction_length, :] for ts in multiple_ts])
test = to_pandasdataset(multiple_ts)
```

The new documentation at https://ts.gluon.ai/stable/tutorials/data_manipulation/pandasdataframes.html#Use-multiple-time-series-for-training states that the multiple_ts was compiled as follows:

```py
multiple_ts = [
    generate_single_ts_with_features(date_range, item_id=i) for i in range(N)
]
```

and the training and test split is done as follows:

```py
to_pandasdataset = lambda data: PandasDataset(
    data,
    feat_dynamic_real=["dynamic_real_1", "dynamic_real_2"],
    feat_static_cat=["static_cat_1"],
)
train = to_pandasdataset([ts.iloc[:-prediction_length, :] for ts in multiple_ts])
test = to_pandasdataset(multiple_ts)
```

Is this (still) ok the way I did it? I did not change the GluonTs version on my PC. 
I can add again that I have already made a lot of forecasts with the procedure I have described and I really hope that this is okay, otherwise I really have a problem.
In this way, the target_value should be predicted using the covariates used. I would be super grateful if someone looks over it. This way really doesn't have to be the most elegant, it just has to be ok 

edit: I just checked the API and I think i have done it correctly, havent I ? Its okay to not give the target item when constructing a list of dataframes and it **is also fine to give the target as the column name in the pandasdataset**, right? I did not write the timestamp, **I do not now for sure but i think then the index is assumed, what do you think?** and i did not give the frequency because my dataframe has the relevant frequency in its structure 

Now i hope that i have done everything okay and right?

**Comment by jaheba** [comment]
Hi, not sure I can follow your code.

Is each `DataFrame` a time series? 

It should suffice to generate one `PandasDataset`, which then can be split.

**Reply by MarleneBs** [reply]
yes each dataframe in a timeseries

I think that i did that with my code, for 
in summary, i did the following to have a split dataset 👍 
to_pandasdataset = lambda data: PandasDataset( #i create the Pandasdataset here
    [dataframe1, dataframe2,...],
    target="vakue1",
    feat_dynamic_real=feat,
)
train = to_pandasdataset([ts.iloc[:-prediction_length, :] for ts in multiple_ts]) # I split the Dataset into training and test 
test = to_pandasdataset(multiple_ts)

**Reply by MarleneBs** [reply]
and in the new tutorials on the page it is a little bit different

**Reply by MarleneBs** [reply]
and i also now there are different ways to do that like with a long dataframe or with the splitter, but I did a lot of work with this code i used and it would be very catastrophic for me if it were **wrong**
